src/components/MyKiva/BadgeModalContentJourney.vue

```diff
@@ -1,97 +1,113 @@
-				:style="{
-					marginTop: `${isMobile || position == 0 ? 0 : (position === 1 ? 100 : 200)}px`,
-					zIndex: positions.length - index,
-					width: '133px',
+					'tw-flex-col tw-py-2 tw-px-2': isMobile,
+					'tw-flex-row tw-py-4 tw-px-0.5': !isMobile,
 				}"
 			>
-				<div class="tw-relative tw-text-center tw-bg-white tw-cursor-pointer" @click="handleBadgeClick(index)">
-					<BadgeContainer :status="getBadgeStatus(index)" :shape="getBadgeShape(badgeWithVisibleTiers.id)">
-						<component
-							v-if="index > 0"
-							:is="getLineComponent(positions[index - 1], position)"
-							class="tw-absolute"
-							:style="getLineStyle(positions[index - 1], position)"
-						/>
-						<img
-							:src="badgeWithVisibleTiers.contentfulData[index].imageUrl"
-							alt="Badge"
-							style="height: 133px; width: 133px;"
-						>
-					</BadgeContainer>
+				<div
+					v-for="(_, index) in positions"
+					:key="index"
+				>
 					<div
-						v-if="showEarnBadge(index)"
-						class="tw-absolute tw-flex tw-items-center tw-justify-center tw-rounded-full tw-min-w-3
-							tw-min-h-3 tw-font-medium tw-bg-gray-200 tw-text-center tw-z-2 tw-px-0.5
-							tw-aspect-square tw-text-small"
-						:style="getNumberCircleStyles()"
+						class="badge tw-relative tw-flex tw-items-center"
+						:style="{
+							marginTop: `${isMobile || index === 0 ? 0 : 75}px`,
+							zIndex: positions.length - index,
+						}"
 					>
-						{{ badgeWithVisibleTiers.achievementData.totalProgressToAchievement }}
-					</div>
-				</div>
-				<div class="tw-text-center tw-bg-white tw-z-1 tw-relative">
-					<div class="tw-font-medium">
-						{{ getTierName(index) }}
-					</div>
-					<div class="tw-text-small">
-						{{ tierCaption(index) }}
+						<div
+							class="tw-relative tw-flex tw-text-center tw-bg-white tw-cursor-pointer"
+							@click="handleBadgeClick(index)"
+						>
+							<BadgeContainer
+								:status="getBadgeStatus(index)"
+								:shape="getBadgeShape(badgeWithVisibleTiers.id)"
+							>
+								<img
+									:src="badgeWithVisibleTiers.contentfulData[index].imageUrl"
+									alt="Badge"
+									style="height: 133px; width: 133px;"
+								>
+							</BadgeContainer>
+						</div>
+						<div class="tw-text-left tw-bg-white tw-z-1 tw-relative tw-px-2 tw-space-y-1">
+							<div class="tw-font-small tw-px-0" v-if="getBadgeStatus(index) !== BADGE_LOCKED">
+								<span v-if="getBadgeStatus(index) == BADGE_COMPLETED">
+									{{ getTierData(index).achievementData.target }}
+									loans {{ badgeSubtitle }}
+								</span>
+								<span v-if="getBadgeStatus(index) == BADGE_IN_PROGRESS">
+									{{ tierSubtitle(index) }}
+								</span>
+							</div>
+							<div
+								class="tw-inline-flex tw-items-center tw-rounded tw-px-1 tw-space-x-1"
+								:class="{'tw-bg-eco-green-1 tw-py-1': getBadgeStatus(index) == BADGE_IN_PROGRESS}"
+							>
+								<kv-icon
+									v-if="(getBadgeStatus(index) == BADGE_IN_PROGRESS)"
```

Comment:
Looks like this is missing the green icon and pill from mocks.

Current:
<img width="235" alt="image" src="https://github.com/user-attachments/assets/0b2cef58-2f4b-42a2-9805-3af233e7f4a9" />

Mocks:
<img width="529" alt="image" src="https://github.com/user-attachments/assets/b4c1f0ae-ab27-440f-b50e-8d6ffc970f2a" />